site/main.go

```diff
@@ -54,6 +58,70 @@ func sub(client mqtt.Client, topic string) {
 	fmt.Printf("Subscribed to topic %s", topic)
 }
 
+/**
+* Create an assessment to analyze the risk of an UI action.
+*
+* @param projectID: GCloud Project ID
+* @param recaptchaSiteKey: Site key obtained by registering a domain/app to use recaptcha services.
+* @param token: The token obtained from the client on passing the recaptchaSiteKey.
+* @param recaptchaAction: Action name corresponding to the token.
+*/
+func createAssessment(projectID string, recaptchaSiteKey string, actionToken string, recaptchaAction string) bool {
+
+	// Create the recaptcha client.
+	// TODO: To avoid memory issues, move this client generation outside
+	// of this example, and cache it (recommended) or call client.close()
+	// before exiting this method.
+	ctx := context.Background()
+	client, err := recaptcha.NewClient(ctx)
+	if err != nil {
+		fmt.Printf("Error creating reCAPTCHA client:%s\n", err)
+		return false
+	}
+	defer client.Close()
```

Comment:
You should follow this TODO.

Reply:
I feel like we're gonna keep using the client, so we might as well keep it around, y'know?